.travis.yml

```diff
@@ -1,14 +1,26 @@
 language: cpp
 matrix:
   include:
-    - name: "ARCH=arm"
-      env: ARCH=arm
-    - name: "ARCH=arm64"
-      env: ARCH=arm64
-    - name: "ARCH=arm64 LD=ld.lld"
-      env: ARCH=arm64 LD=ld.lld-8
-    - name: "ARCH=x86_64"
-      env: ARCH=x86_64
+    - name: "ARCH=arm REPO=linux"
+      env: ARCH=arm REPO=linux
```

Comment:
Rather than explicitly stating `REPO=linux`, can we just leave it out, since it's the default? Similar to what we do for bfd vs lld for LD?  I think this makes the names a little bit more readable, too.

Reply:
(reposting since replying via email doesn't keep comments inline)

Yes fair point, done: ee9c829c3d8527fd3afc93ab8c165f81db108658

Turns out this might be beneficial since adding a new variable appears to create a
new cache...